model/src/main/java/org/cbioportal/genome_nexus/model/PdbHeader.java

```diff
@@ -0,0 +1,54 @@
+package org.cbioportal.genome_nexus.model;
+
+import java.util.Map;
+
+/**
+ * @author Selcuk Onur Sumer
+ */
+public class PdbHeader
+{
+    private String pdbId;
+    private String title;
+    private Map<String, Object> compound;
+    private Map<String, Object> source;
```

Comment:
Keeping this not-so-nice model as is for now in order to avoid front-end changes. We may want to revisit this model if we switch to a better web service in the future (XML or JSON).